packages/query-core/src/tests/queryClient.test.tsx

```diff
@@ -124,6 +124,8 @@ describe('queryClient', () => {
     })
 
     test('should warn in dev if several query defaults match a given key', () => {
```

Comment:
This test keeps on failing as the console to called lesser times than expected.

Reply:
probably because we don't have this warning anymore?

https://github.com/TanStack/query/pull/4721/files/09fe1ea6045cb9495841f8c782a7da78715122f0#diff-bf605a2d8d2cf086c1781865ec3cb71ebc3726f953e6f992d520978690948bfcR224-R227

maybe we need to just decrease the count by one?